crates/hir_def/src/attr.rs

```diff
@@ -171,13 +171,19 @@ pub struct AttrQuery<'a> {
 }
 
 impl<'a> AttrQuery<'a> {
+    /// For an attribute like `#[attr(value)]`, returns the `(value)` subtree.
+    ///
+    /// If the attribute does not have a token tree argument, returns `None`.
```

Comment:
Returns an empty subtree?

Reply:
Returns `None` for attributes without argument, or with an `= "literal"` argument is what I meant, I'll reword

Reply:
But the function returns an iterator, not an Option?

Reply:
Oh, duh. Sorry

Reply:
That API is confusing, I've remove the comments again